stripe-core/res/values-de/strings.xml

```diff
@@ -10,6 +10,8 @@
   <string name="address_label_country_or_region">Land oder Region</string>
   <!-- Caption for County field on address form (only countries that use county, like United Kingdom) -->
   <string name="address_label_county">Kreis/Bezirk</string>
+  <!-- Placeholder string for name entry field. -->
+  <string name="address_label_full_name">Nachname</string>
```

Comment:
This is incorrect — it says “last name” instead of “full name”. How are we sourcing these translations?

Reply:
It is coming from lokalise, our translating service

Reply:
Updated the DE translation!

Reply:
Thanks James — my people appreciate it 😉